src/frontend/components/modal/modals/curval/lib/component.js

```diff
@@ -330,14 +333,13 @@ class CurvalModalComponent extends ModalComponent {
 
   }
 
-  getURL(current_id, instance_name, layout_id, form_data) {
+  getURL(current_id, instance_name, layout_id) {
 
     let url = current_id
       ? `/record/${current_id}`
       : `/${instance_name}/record/`
 
-    url = `${url}?include_draft&modal=${layout_id}`
-    if (form_data) url = url + `&${form_data}`
+    url += `?csrf-token=${$('body').data("csrf")}&modal=${layout_id}&include_draft=1`
```

Comment:
I don't think the CSRF token needs to be in the query - it should suffice just in the body (as presumably it already is)